charts/fluentd/templates/_helpers.tpl

```diff
@@ -74,4 +74,8 @@ Builds the list for exclude paths in the tail for the containers - windows
 {{- else }}
 {{- print .Values.windowsDaemonset.excludeFluentdPath }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- define "logzio.secretName" }}
+{{- print .Values.secretName }}
+{{- end -}}
```

Comment:
This function now does the same thing as using `{{ .Values.secretName }}` directly.